content/yaml/yaml-getting-started.md

```diff
@@ -228,10 +238,9 @@ You can freely use all of the above features of environment section in conjuncti
 
 ```yaml
 environment:
-  vars: # Define your environment variables here
+  vars: # Define your public environment variables here
     PUBLIC_ENV_VAR: 'value here'
```

Comment:
Can we use double quotes for consistency?

Reply:
suggested change has been made. Kindly approve it